src/transformers/configuration_longformer.py

```diff
@@ -42,7 +42,7 @@ class LongformerConfig(RobertaConfig):
     It reuses the same defaults. Please check the parent class for more information.
 
     Args:
-        attention_window (:obj:`int` or :obj:`List[int]`, optional, defaults to 512):
+        attention_window (:obj:`int` or :obj:`List[int]`, `optional`, defaults to 512):
```

Comment:
Collecting cookie points @sgugger

Reply:
I have a massive docs PR I'm actually preparing for those ;-)

Reply:
Noticed that with some merge conflicts :D

Reply:
I don't know what you mean, it only touched 165 files 😇